client/src/app/metadata-picker/metadata-picker.component.css

```diff
@@ -0,0 +1,6 @@
+.mat-option {
+  height: 200px;
```

Comment:
Not that it matters too much, but rem values [are better](https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/?ref=heydesigner)